packages/utilities/psammead-styles/index.test.jsx

```diff
@@ -74,6 +74,7 @@ const coloursExpectedExports = {
   C_WEATHER_BLUE: 'string',
   C_ARCHIVE_BLUE: 'string',
   C_KINGFISHER: 'string',
+  C_LEARNINGENGLISH: 'string',
```

Comment:
Can you please give this a _colour_ name rather than a service name? e.g. C_TEAL. I say this because in future if Learning English want to change their colour, we should be able to switch easily and constants like colours should retain their value over time.